### PR TITLE
F#src 175 hand h finder

### PR DESCRIPTION
**Pull request by dg-shadow**

### CODE REVIEW

File: sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py [hunk 1]
```diff
@@ -13,41 +13,23 @@
...
 
         elif hand_or_arm == 'hand':
-            if self.__hand_h:
-                self.__commander = SrRobotCommander("hand_h", prefix="H1_")
-            else:
-                hand_finder = HandFinder()
-
-                hand_parameters = hand_finder.get_hand_parameters()
-                hand_serial = hand_parameters.mapping.keys()[0]
+            self.__commander = SrHandCommander()
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_robot_state_saver.py:27]: Cool to simplify this so much :)

File: sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py [hunk 2]
```diff
@@ -39,7 +40,18 @@ def __init__(self, name=None, prefix=None, hand_parameters=None, hand_serial=Non
...
+        self._hand_h = False
+        if name is None and prefix is None and hand_parameters is None and hand_serial is None:
+            hand_finder = HandFinder()
+            if hand_finder.hand_e_available():
+                name, prefix, hand_serial = hand_finder.get_first_hand_e()
+            elif hand_finder.hand_h_available():
+                self._hand_h = True
+                name, prefix, hand_serial = hand_finder.get_first_hand_h()
+            else:
+                rospy.log_fatal("No way to define hand commander. Throw me a frickin' bone here people...")
```
Comment (toliver) [inline sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py]: I think this should be rephrased.

**Review by toliver — CHANGES_REQUESTED** [review]

File: sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py (hunk 2 above)
Comment (dg-shadow) [reply sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py]: Booooring :-P

Comment (dg-shadow) [reply sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py]: If I make the error message more informative, can I keep the Dr Evil quote?

Comment (toliver) [reply sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py]: I will let you ask Armando about these colorful customer facing messages.

Comment (dg-shadow) [reply sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py]: Laaaaaaaaaame.

Comment (dg-shadow) [reply sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py]: I'll let you ask a dictionary about how to spell colourful.

**Review by toliver — APPROVED** [review]

**Pull request merged**